charts/Makefile

```diff
@@ -22,14 +24,18 @@ vault:
 	@echo "Installing vault ..."
 	kubectl create namespace m4d-system || true
 	helm repo add hashicorp https://helm.releases.hashicorp.com
-	helm install vault hashicorp/vault --version 0.8.0 \
-                --set "global.openshift=false" \
+	helm install vault hashicorp/vault --version 0.9.1 \
+                --set "global.openshift=$(WITHOUT_OPENSHIFT)" \
                 --set "server.dev.enabled=true" \
+		--values ../third_party/vault/values/plugin-secrets-kubernetes-reader.yaml \
```

Comment:
Did you check that it creates the spec you expected? It's unclear from the Helm docs if -f (--values) and --set can be used together. I didn't check.

Reply:
Yes it seems to work ok: the server starts in dev mode and the plugin is enabled.